remaining packer content (#13285)
hashicorp/packer: website/content/docs/communicators/winrm.mdx

---
description: |

>   The `winrm` communication establishes a WinRM connection so that Packer can perform actions, such as upload files and execute scripts, on the machine it creates. 

Reviewer: ```suggestion
  The `winrm` communicator establishes a WinRM connection, letting Packer upload files or execute scripts on the machine it creates. 
```
